tests/pages/services/ServiceSearchFilter-cy.js

```diff
@@ -1,11 +1,21 @@
-xdescribe('Service Search Filters', function () {
+describe('Service Search Filters', function () {
+
   context('Filters services table', function () {
+
+    function getSearchParameter(hash) {
```

Comment:
Let's move this into our test utils and add a few unit test to verify that is actually finds the search parameter.